### PR TITLE
Release 1.3.0

### PR DESCRIPTION
**Pull request by Firstyear**
Release 1.3.0 - this is the prep stage to create the branches.

Checklist

- [ x ] This PR contains no AI generated code
- [ x ] `cargo fmt` has been run
- [ ] `cargo clippy` has been run
- [ x ] `cargo test` has been run and passes
- [ ] book chapter included (if relevant)
- [ ] design document included (if relevant)

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -14,6 +14,52 @@ report it to our [issue tracker].
...
+as this may have important effects on your distribution or upgrades in future.
+
+Before upgrading you should review [our upgrade documentation](https://kanidm.github.io/kanidm/stable/server_updates.html#general-update-notes)
+
+### 1.3.0 Important Changes
+
+- New GID number constraints are now enforced in this version. To upgrade from 1.2.0 all accounts
+  and groups must adhere to these rules. See [our upgrade documentation].
+  about tools to help you detect and correct affected entries.
+- Oauth2 URI require stricter matching rules to be applied from 1.4.0.
```
Comment (yaleman) [inline RELEASE_NOTES.md:34]:
```suggestion
- OAuth2 URIs require stricter matching rules to be applied from 1.4.0.
```

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -14,6 +14,52 @@ report it to our [issue tracker].
...
+This is the latest stable release of the Kanidm Identity Management project. Every release is
+the combined effort of our community and we appreciate their invaluable contributions, comments,
+questions, feedback and support.
+
+You should review our
+[support documentation](https://github.com/kanidm/kanidm/blob/master/book/src/support.md)
+as this may have important effects on your distribution or upgrades in future.
+
+Before upgrading you should review [our upgrade documentation](https://kanidm.github.io/kanidm/stable/server_updates.html#general-update-notes)
+
```
Comment (yaleman) [inline RELEASE_NOTES.md:28]: We should stick to a single linking strategy - one links to the book source in master, the other links to the compiled book in stable? 😄

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -14,6 +14,52 @@ report it to our [issue tracker].
...
+- Reduced memory usage
+- Improved update flow when changing from dev to stable server versions
+- PIV authentication foundations
+- Significant improvements to performance for write and search operations
+- Support Illumos
+- Begin rewrite of the webui
+- Oauth2 allows multiple origins
+- Lengthen replication mtls certificate life
+- Unixd allows home paths to be in an external mount folder
+- Strict redirect URI enforcement in Oauth2
```
Comment (yaleman) [inline RELEASE_NOTES.md:60]:
```suggestion
- OAuth2 allows multiple origins
- Lengthen replication MTLS certificate lifetime
- UNIX daemon allows home paths to be in an external mount folder
- Strict redirect URI enforcement in OAuth2
```

**Review by yaleman — CHANGES_REQUESTED** [review]
minor nits but woooo 1.3.0

File: RELEASE_NOTES.md (hunk 2 above)
Comment (Firstyear) [reply RELEASE_NOTES.md:28]: Oopppp,s good point.

**Review by yaleman — APPROVED** [review]

**Pull request merged**